pages/about.html

```diff
@@ -4,9 +4,47 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Olive Lavine</title>
+  <link rel="stylesheet" href="/stylesheets/style.css">
 </head>
 <body>
-  
+  <header>
+    <img src="/images/olive.jpg" alt="A black and white photo of a person 
+    from the shoulders up, with pale skin and straight long hair in a graduation 
+    cap and gown, against a background of clear skies and a tall city building" />
+    <h1>Olive Henzel Lavine</h1>
+    <nav>
+      <menu>
+        <li><a href="/pages/index.html">Home</a></li>
+        <li><a href="/pages/portfolio.html">Portfolio</a></li>
+        <li><a href="#">About Me</a></li>
+      </menu>
+    </nav>
+  </header>
+  <main>
+    <section>
+      <h2>Personal</h2>
+      <p>lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras ut feugiat tellus, nec sagittis augue.
+        Donec et augue mollis, feugiat augue porttitor, imperdiet metus. Nam ut consequat ante, sed mattis ipsum.
+        Quisque porttitor tellus eget orci sodales, id maximus orci commodo.
+        </p>
+    </section>
+    <section>
+      <h2>Professional</h2>
+      <p>lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras ut feugiat tellus, nec sagittis augue.
+        Donec et augue mollis, feugiat augue porttitor, imperdiet metus. Nam ut consequat ante, sed mattis ipsum.
+        Quisque porttitor tellus eget orci sodales, id maximus orci commodo.
+        </p>
+    </section>
+  </main>
+  <footer>
+    <nav>
```

Comment:
Nice use of `nav` to wrap the footer since it's made up of links to various locations.